p2p/antiflood/p2pAntiflood_test.go

```diff
@@ -0,0 +1,134 @@
+package antiflood_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/check"
+	"github.com/ElrondNetwork/elrond-go/p2p"
+	"github.com/ElrondNetwork/elrond-go/p2p/antiflood"
+	"github.com/ElrondNetwork/elrond-go/p2p/mock"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewP2pAntiflood_NilFloodPreventerShouldErr(t *testing.T) {
+	t.Parallel()
+
+	afm, err := antiflood.NewP2pAntiflood(nil)
+
+	assert.True(t, check.IfNil(afm))
+	assert.True(t, errors.Is(err, p2p.ErrNilFloodPreventer))
+}
+
+func TestNewP2pAntiflood_ShouldWork(t *testing.T) {
+	t.Parallel()
+
+	afm, err := antiflood.NewP2pAntiflood(&mock.FloodPreventerStub{})
+
+	assert.False(t, check.IfNil(afm))
+	assert.Nil(t, err)
+}
+
+func TestP2pAntiflood_SettingInnerFloodPreventerToNil(t *testing.T) {
+	t.Parallel()
+
+	afm, _ := antiflood.NewP2pAntiflood(&mock.FloodPreventerStub{})
+	afm.FloodPreventer = nil
+
+	assert.True(t, check.IfNil(afm))
+}
+
+//------- CanProcessMessage
+
+func TestP2pAntiflood_CanProcessMessageNilFloodPreventerShouldError(t *testing.T) {
+	t.Parallel()
+
+	afm, _ := antiflood.NewP2pAntiflood(&mock.FloodPreventerStub{})
+	afm.FloodPreventer = nil
+
+	err := afm.CanProcessMessage(&mock.P2PMessageMock{}, "connected peer")
+
+	assert.Equal(t, p2p.ErrNilFloodPreventer, err)
+}
+
+func TestP2pAntiflood_CanProcessMessageNilMessageShouldError(t *testing.T) {
+	t.Parallel()
+
+	afm, _ := antiflood.NewP2pAntiflood(&mock.FloodPreventerStub{})
+
+	err := afm.CanProcessMessage(nil, "connected peer")
+
```

Comment:
remove empty lines (in the entire file between err and assert error)

Reply:
done